src/Grids/grid_timescales.jl

```diff
@@ -0,0 +1,18 @@
+
+#####
+##### Getting minima of grid Δs 
+#####
+
+min_Δxyz(grid) = min(min_Δx(grid), min_Δy(grid), min_Δz(grid))
+min_Δxy(grid) = min(min_Δx(grid), min_Δy(grid))
+
+min_Δx(grid) = grid.Δx
+min_Δy(grid) = grid.Δy
+min_Δz(grid) = grid.Δz
```

Comment:
These should dispatch on `RegularRectilinearGrid` since they won't work for any other grid type.

Reply:
In addition, it may be better to move these to `regular_rectilinear_grid.jl`.